### ISSUE TITLE
Support cbindgen config

### ISSUE THREAD

**Issue by Roms1383** [thlorenz/rid#58]
Hello ! I tried something a bit out of scope but still:

## context

Use conditional compiling in `rid` plugin crate, gated behind a `feature flag`.

## steps to reproduce

Just use a feature flag somewhere in your code to gate some module, e.g. :
```rust
// lib.rs
#[cfg(feature = "flutter")]
pub mod plugin; // my plugin with rid store, etc
```

## issue encountered

Not an error in itself, more like a `cbindgen` warning, e.g. :
```sh
2021-11-09 16:05:35,132 WARN [cbindgen::bindgen::ir::cfg] Missing `[defines]` entry for `feature = "flutter"` in cbindgen config.
```
> whenever running ./sh/bindgen

So I looked on internet and found out more informations in `cbindgen` docs [under the section "Defines and Cfgs"](https://github.com/eqrion/cbindgen/blob/master/docs.md#defines-and-cfgs).

So naturally I went to look for mentions of it in `rid` sourcecode and ended up landing on `rid-build`'s [fn cbingen()](https://github.com/thlorenz/rid/blob/c5c8fdda7aaad33e0ee36fa12be9c777ec1ba58c/rid-build/src/bindings_generator.rs#L49) which configures `cbindgen` `Builder` and naturally the [fn with_define()](https://docs.rs/cbindgen/0.20.0/cbindgen/struct.Builder.html#method.with_define) caught my attention: I gave a quick try on [a fork's branch of mine](https://github.com/Roms1383/rid/tree/chore/cbindgen-defines), and it works.

## summary

Long story short, I'm not gonna open a PR since these changes are only relevant to my own setup and my own project.
But I thought worth mentioning that in the future maybe people would like the ability to tweak `cbindgen` config.

I looked at it briefly, and apologies if there's some existing mechanism to do it already:
then I think it would be worth mentioning it in `rid` docs.

Thanks  🐘
